charts/promtail/values.yaml

```diff
@@ -278,10 +283,19 @@ config:
             source_labels:
               - __meta_kubernetes_pod_label_app_kubernetes_io_name
             target_label: app
+          - action: drop
+            regex: ''
+            source_labels:
+              - app
```

Comment:
are these changes intended? I don't see how this relates to the `addScrapeJobLabel` option.

Reply:
Yes, that's what this PR is mostly about. We need to drop out here when there is no app label in order to avoid duplicates with an incomplete job label. These pods will be covered by one of the other scrape jobs.

That's not related to `addScrapeJobLabel`.